### PR TITLE
RENO-2916: CardGrid Test

### PR DESCRIPTION
**Pull request by isastettler**
[Jira Ticket](https://jira.nypl.org/browse/RENO-2916)

**This PR does the following:**
- add tests for CardGrid.tsx and Card.tsx

### Review Steps:
- [ ] pull in branch `git pull origin RENO-2916/cardgrid-test`
- [ ] run `npm test`

### CODE REVIEW

File: src/components/home-v1/mockContent.js [hunk 1]
```diff
@@ -75,6 +78,7 @@ export const featuredEvents = {
           url: "https://www.showclix.com/event/thetrayvongeneration/tag/nyplhomepage",
         },
         {
+          id: "aurthor-talks-4",
```
Comment (nypl-wluisi) [inline src/components/home-v1/mockContent.js:81]: "author"

Comment (nypl-wluisi) [inline src/components/home-v1/mockContent.js:81]: "author"

File: src/components/home-v1/mockContent.js [hunk 2]
```diff
@@ -131,20 +140,23 @@ export const featuredEvents = {
           url: "https://www.nypl.org/events/calendar?keyword=NoNieqa+Ramos",
         },
         {
+          id: "kids-teens-2",
           title: "NYPL After School",
           date: "MONDAY–THURSDAY",
```
Comment (nypl-wluisi) [inline src/components/home-v1/mockContent.js:145]: For this and other date values, have them here as regular case, so "Monday-Thursday" or "Ongoing" and then cast them to all uppercase using CSS.

Comment (nypl-wluisi) [inline src/components/home-v1/mockContent.js:145]: For this and other date values, have them here as regular case, so "Monday-Thursday" or "Ongoing" and then cast them to all uppercase using CSS.

**Review by nypl-wluisi — CHANGES_REQUESTED** [review]
@isastettler In general looks great, just a few changes.

File: src/components/home-v1/CardGrid/Card.tsx [hunk 3]
```diff
@@ -38,35 +38,60 @@ interface CardProps {
...
   return (
     <Grid
       templateRows={{ base: "1fr", md: "min-content" }}
       templateColumns={{ base: "5fr 9fr", md: "1fr" }}
       sx={styles}
     >
       <GridItem className="textBox">
-        <Heading as="h3" aria-description={ariaDescription}>
-          <Link href={item.url}>{item.title} </Link>
+        <Heading as="h3" aria-describedby={describedByString}>
```
[the remark this answers is not in the document]
Comment (isastettler) [reply src/components/home-v1/CardGrid/Card.tsx:57]: Oh this is awesome! I tried to do something like that and could not get it to work!!

File: src/components/home-v1/mockContent.js (hunk 2 above)
Comment (isastettler) [reply src/components/home-v1/mockContent.js:145]: I changed the data, the css was already set up to change it to all uppercase.

**Comment by nypl-wluisi** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr276-twuv78uqgl1vjyrw9lfpbypnbndvashs.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/627a9f38c4a9adf3431d34f5

**Review by nypl-wluisi — APPROVED** [review]

**Pull request merged**